elado/birtok/index.md

```diff
@@ -0,0 +1,73 @@
+
+Az épület fenntartása gazdaságos módon történik. A 70 m mély fúrt kút ivóvíz minőségű vize biztosítja a lakók vízszükségletét, a meleg vizet két gázbojler állítja elő. A keletkező szennyvizet az engedély alapján készült szikkasztóba vezetik. A fűtést is segítheti a gáz, ám a birtokról mindig származik annyi kitermelhető fa, amellyel a tervezett kandalló vagy a nagy teljesítményű vegyes tüzelésű kazán meleget ad az otthonnak.
+
+A kereszt alakú lakóház alagsorában található a kazánház a két kazánnal, pince, nyári konyha és csónaktároló. A földszinten amerikai konyha, 50 m2-es nappali szoba, 3 hálószoba, 1 dolgozó szoba, 1 vendégszoba, személyzeti szoba, 3 fürdőszoba, télikert, közlekedő, kétállásos fűtött garázs és hatalmas teraszok vannak. A lakóház része egy külön bejáratú, önálló, teraszos, egyszobás apartman is. 
+
+A kb. 80 m2-es személyzeti lakás különálló kis épületben biztosít barátságos otthont a természettisztelő segítőknek.
+
+Sétálni, futni, biciklizni, akár lovagolni, csónakázni, vitorlázni, szörfözni, horgászni, kertészkedni, kedvenc állatot tartani testi-lelki egészségünk érdekében, mindez birtokon belül lehetséges! Minden évszakban és minden korosztálynak megnyugtató és egyben felkavaró, izgalmas és békés helyszín ez a családi birtok, de talán a gyermekneveléshez adja a legtöbb és legfontosabb segítséget. 
+
+Az ingatlant adottságai alkalmassá teszik akár üzleti célú felhasználásra is. A természet megóvása mellett még további lakóházak is építhetők a tavak prtjára. Változatos helyszíne filmforgatásra is alkalmassá teszi, sőt, ihletet is nyújtó, de pl. közösségépítő-, testi-lelki rehabilitációs-, speciális idős- vagy gyermekotthon munkáját is jelentősen támogató tényező, hogy e varázslatos birtokon a természet részeként élhetünk.
+
+Budapesttől és a schwechati reptértől 120 km-re bújik meg. 
+Az irányára kilencszázötvenezer Euro, vagy ennek megfelelő HUF.
+További információkat kérni, vagy a megtekintéshez időpontot egyeztetni az oldalunkon ismertetett elérhetőségeken lehet.
+Várom megkeresését!
+
+Tisztelettel: 	Darabosné Zsóka :)
+
+# Tavasz a birtokon
+
+#![Kívülről](http://i.imgur.com/CAniLQi.jpg)
+#![Kívülről](http://i.imgur.com/DIfS6Ma.jpg)
+#![Kívülről](http://i.imgur.com/tFBaYDV.jpg)
+
+# Nyár
+
+#![Kívülről](http://i.imgur.com/qlsJhDo.jpg)
+#![Kívülről](http://i.imgur.com/abvYTQw.jpg)
+#![Kívülről](http://i.imgur.com/Y74UV5D.jpg)
+
+# Ősz
+
+#![Kívülről](http://i.imgur.com/Tw5fRtU.jpg)
+#![Kívülről](http://i.imgur.com/lQApcH0.jpg)
+#![Kívülről](http://i.imgur.com/mCprXvQ.jpg)
+#![Kívülről](http://i.imgur.com/0YieXek.jpg)
+# Személyzeti épület
+#![Kívülről](http://i.imgur.com/slWbkaF.jpg)
+
+# Tél
+
+#![Kívülről](http://i.imgur.com/rXBlorY.jpg)
+#![Kívülről](http://i.imgur.com/a8933Ir.jpg)
+#![Kívülről](http://i.imgur.com/9sKWRit.jpg)
+#![Kívülről](http://i.imgur.com/rTDtkVj.jpg)
+#![Kívülről](http://i.imgur.com/hzfLpmK.jpg)
+#![Kívülről](http://i.imgur.com/UKl84dt.jpg)
+#![Kívülről](http://i.imgur.com/NEaUyFO.jpg)
```

Comment:
igen, ezen gondolkodtam is, hogy lecserélem valamire... csak kevés a belső
fotó...
Majd talán holnap.
Remélem, meg tudom majd csinálni a malmot is :)
Nem tudom, miért nem találok soha vissza oda, ahová szeretnék...

2015. január 12. 0:39 Daniel Darabos írta, <notifications@github.com>:

> In elado/birtok/index.md
> <https://github.com/darabos/megingat/pull/4#discussion-diff-22771908>:
>
> > +#![Kívülről](http://i.imgur.com/Tw5fRtU.jpg)
> > +#![Kívülről](http://i.imgur.com/lQApcH0.jpg)
> > +#![Kívülről](http://i.imgur.com/mCprXvQ.jpg)
> > +#![Kívülről](http://i.imgur.com/0YieXek.jpg)
> > +# Személyzeti épület
> > +#![Kívülről](http://i.imgur.com/slWbkaF.jpg)
> > +
> > +# Tél
> > +
> > +#![Kívülről](http://i.imgur.com/rXBlorY.jpg)
> > +#![Kívülről](http://i.imgur.com/a8933Ir.jpg)
> > +#![Kívülről](http://i.imgur.com/9sKWRit.jpg)
> > +#![Kívülről](http://i.imgur.com/rTDtkVj.jpg)
> > +#![Kívülről](http://i.imgur.com/hzfLpmK.jpg)
> > +#![Kívülről](http://i.imgur.com/UKl84dt.jpg)
> > +#![Kívülről](http://i.imgur.com/NEaUyFO.jpg)
>
> Ez az utolsó kép nekem úgy tűnik, hogy nem olyan jó felbontású, mint a
> többi.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/darabos/megingat/pull/4/files#r22771908>.
>